tests/baselines/reference/api/typescript.d.ts

```diff
@@ -2821,8 +2821,8 @@ declare namespace ts {
             private hasAddedOrRemovedSymlinks;
             protected languageService: LanguageService;
             languageServiceEnabled: boolean;
-            readonly trace?: (s: string) => void;
-            readonly realpath?: (path: string) => string;
+            trace?(s: string): void;
+            realpath?(path: string): string;
```

Comment:
I think these need to be updated (noticed the change without baseline update).

(Not sure if any other ones can be reverted either)